docs/concepts/models.md

```diff
@@ -885,63 +809,109 @@ except ValidationError as e:
-from typing import Generic, TypeVar
+    ```py test="skip" lint="skip"
+    class MyIntModel(MyGenericModel[int]): ...
+
+    isinstance(my_model, MyIntModel)
+    ```
+
+### Validation of unparametrized type variables
+
+When leaving type variables unparametrized, Pydantic treats generic models similarly to how it treats built-in generic
+types like [`list`][] and [`dict`][]:
+
+* If the type variable is [bound](https://typing.readthedocs.io/en/latest/reference/generics.html#type-variables-with-upper-bounds)
+  or [constrained](https://typing.readthedocs.io/en/latest/reference/generics.html#type-variables-with-constraints) to a specific type,
+  it will be used.
+* If the type variable has a default type (as specified by [PEP 696](https://peps.python.org/pep-0696/)), it will be used.
+* For unbound or unconstrained type variables, Pydantic will fallback to [`Any`][typing.Any].
+
+```py
+from typing import Generic
+
+from typing_extensions import TypeVar
 
 from pydantic import BaseModel, ValidationError
 
-AT = TypeVar('AT')
-BT = TypeVar('BT')
+T = TypeVar('T')
+U = TypeVar('U', bound=int)
+V = TypeVar('V', default=str)
 
 
-class Model(BaseModel, Generic[AT, BT]):
-    a: AT
-    b: BT
+class Model(BaseModel, Generic[T, U, V]):
+    t: T
+    u: U
+    v: V
 
 
-print(Model(a='a', b='a'))
-#> a='a' b='a'
+print(Model(t='t', u=1, v='v'))
+#> t='t' u=1 v='v'
 
-IntT = TypeVar('IntT', bound=int)
-typevar_model = Model[int, IntT]
-print(typevar_model(a=1, b=1))
-#> a=1 b=1
 try:
-    typevar_model(a='a', b='a')
+    Model(t='t', u='u', v=1)
 except ValidationError as exc:
     print(exc)
     """
-    2 validation errors for Model[int, TypeVar]
-    a
-      Input should be a valid integer, unable to parse string as an integer [type=int_parsing, input_value='a', input_type=str]
-    b
-      Input should be a valid integer, unable to parse string as an integer [type=int_parsing, input_value='a', input_type=str]
+    2 validation errors for Model
+    u
+      Input should be a valid integer, unable to parse string as an integer [type=int_parsing, input_value='u', input_type=str]
+    v
+      Input should be a valid string [type=string_type, input_value=1, input_type=int]
     """
-
-concrete_model = typevar_model[int]
-print(concrete_model(a=1, b=1))
-#> a=1 b=1
 ```
 
 !!! warning
-    While it may not raise an error, we strongly advise against using parametrized generics in isinstance checks.
 
-    For example, you should not do `isinstance(my_model, MyGenericModel[int])`. However, it is fine to do `isinstance(my_model, MyGenericModel)`. (Note that, for standard generics, it would raise an error to do a subclass check with a parameterized generic.)
+    If a subtype of the type variable upper bound, constraints or default is being used and the model isn't explicitly parametrized, the resulting type **will not be** the one being provided. In the case of type variables bound to Pydantic models, this can lead to a data loss:
```

Comment:
```suggestion
    If a subtype of the type variable upper bound, constraints, or default is being used and the model isn't explicitly parametrized, the resulting type **will not be** the one being provided. In the case of type variables bound to Pydantic models, this can lead to a data loss:
```

Oxford comma adds clarity here ^^

This sentence is a bit long, what if we simplified to:

In some cases, validation against an unparametrized generic model can lead to data loss. Specifically, if a subtype of the type variable upper bound, constraints, or default is validated, ...